pyacq/devices/tests/test_eeg_emotiv.py

```diff
@@ -0,0 +1,81 @@
+import time
+import pytest
+
+from pyacq.devices.eeg_emotiv import Emotiv, HAVE_PYCRYPTO  # , HAVE_PYWINUSB
+from pyacq.viewers.qoscilloscope import QOscilloscope
+from pyqtgraph.Qt import QtCore, QtGui
+
+from collections import OrderedDict
+import os
+
+
+# Manual scan of devices
+def get_available_devices():
```

Comment:
This is a generally useful function. I suggest making it a class method of `Emotiv`.